Python/Tkinter/Calculator/calculator.py

```diff
@@ -0,0 +1,122 @@
+# ===========================================================================
+# Title: calculator.py
+# Description: This program is a calculator
+# Author : Mathieu Caro
+# Source : Udemy
+# ===========================================================================
+
+# -----------------------------Import of modules------------------------------
+import tkinter as tk  # tkinter: package for standard python interface
+import tkinter.font as font
+from tkinter import ttk  # tkk: set of widgets:buttons, labels...
+# ----------------------------------Constants---------------------------------
+ORANGE = "#FF7900"
+BLACK = "#000000"
+WHITE = "#FFFFFF"
+BLUE = "#0090FC"
+# -----------------------------Global Variables--------------------------------
+equation=""
+# ---------------------------------Functions----------------------------------
+def show(value):
+    """This function enables to set the value of each button on screen"""
+    global equation
+    equation+=value #we add each button value to the string equation
+    label_result.set(equation) 
+    #for each additional button value it updates the value of the equation
+    
+def clear():
+    """This function simply clear the equation after using the "C" button"""
```

Comment:
some time you use # for a commentary and """ you should use # for a commentary on 1 line and use """ for multiline commentary